recipe/meta.yaml

```diff
@@ -1,29 +1,34 @@
+{% set name = "jupyter_bokeh" %}
 {% set version = "3.0.7" %}
 
 package:
-  name: jupyter_bokeh
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/j/jupyter_bokeh/jupyter_bokeh-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
   sha256: 2da8c3ddc734d15737bf06126d9e31e84d30f18ac3da3a3f95be40a95a054c87
 
 build:
-  number: 0
-  noarch: python
-  script: {{ PYTHON }} -m pip install . -vv --install-option="--skip-npm"
+  number: 1
+  skip: True # [s390x or  py<38]
+  script: {{ PYTHON }} -m pip install . -vv --install-option="--skip-npm" --no-deps
 
 requirements:
+  build:
+    - {{ compiler('c') }}
   host:
-    - jupyter-packaging >=0.7.9,<0.8
+    - jupyter-packaging 0.7.9
     - pip
-    - python >=3.8
+    - python
+    - setuptools
+    - wheel
   run:
-    - bokeh >=3,<4
+    - bokeh >=3,<4 
     - ipywidgets >=8,<9
-    - python >=3.8
-  run_constrained:
+    - python
     - jupyterlab >=3,<4
```

Comment:
I think jupyterlab should be moved to host section as the pyproject.toml file has these four together in build-system:
"jupyter_packaging~=0.7.9", "jupyterlab~=3.0", "setuptools>=40.8.0", "wheel"

Reply:
done

Reply:
It's better to include `jupyterlab` to run_constrained, see https://github.com/conda-forge/jupyter_bokeh-feedstock/commit/c48494fc52f52595348608971cc95e6352c72098

Reply:
@skupr-anaconda but why, given it lives in build in the upstream:

https://github.com/bokeh/jupyter_bokeh/blob/b98f3c70a16b5d7b33072dedbc4d6ab4e10faa59/pyproject.toml

Leaving jupyterlab in run_constrained allows to build it even when jupyterlab is not available on the platform (s390x), but then it fails as it is used during runtime.

Reply:
Okay I understand why we don't need it in the host section, my bad. But I think we need it in the run section, otherwise we might build on the platform we can't test for?